daprdocs/content/en/reference/components-reference/supported-secret-stores/kubernetes-secret-store.md

```diff
@@ -7,9 +7,27 @@ aliases:
   - "/operations/components/setup-secret-store/supported-secret-stores/kubernetes-secret-store/"
 ---
 
```

Comment:
Need to add a header & section on the built-in Kubernetes secret store, why it's there, and when users should use it vs when they should not.